CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.16)
+cmake_policy(VERSION 3.12)
+project(KineticGas LANGUAGES C CXX)
+
+SET(CMAKE_CXX_STANDARD 17)
+
+if(NOT MSVC)
+  # set(CMAKE_CXX_COMPILER /usr/bin/g++) # Setting these can, on some systems, cause cmake to go into an infinite loop (see: build.sh to set the compiler)
+  # set(CMAKE_C_COMPILER /usr/bin/gcc) # Setting these can, on some systems, cause cmake to go into an infinite loop (see: build.sh to set the compiler)
+  if(APPLE)
+    set(CMAKE_CXX_FLAGS "-Wall -Wextra -Wno-unknown-pragmas -Wno-sign-compare -pthread -arch arm64")
```

Comment:
Ooops this should be fixed. The arch should not be specified this way. It should instead be something like: https://github.com/usnistgov/teqp/blob/201ceb7c59780bd93df484b90c2b2e986823116f/CMakeLists.txt#L2